quizzes/quiz_ch2.md

```diff
@@ -0,0 +1,32 @@
+
+{quiz, id: reproducible_code, attempts: 10}
+
+## What is R?
+
+? What is R?
+
+A) A language for working with data
+b) A set of code that allows you to do various things
+c) A type of programming notation
+d) A language for creating websites
+
+? What does "open source" mean?
+
+a) The code is only available to a select few users
+b) The code is hidden and can't be accessed by users
+C) The code is publicly available and others are able to contribute to it
+d) The code is open for viewing but not modification
+
+? What is an R package?
+a) A way to make your code more complicated
+b) A way to protect your code from being seen by others
+c) A type of statistical analysis
+D) A set of code that can be shared between users
+
+? What is one benefit of having a rich and active community in R?
+a) More opportunities to make your code more complicated
+b) Less support for users
+C) More resources for finding help and tutorials
+d) More difficult to find others who use R
+
```

Comment:
Maybe we could add questions about why R is good for reproducibility?

Reply:
Good call